src/index.js

```diff
@@ -9,4 +9,29 @@ function component() {
   return element;
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+document.body.appendChild(component());
+
+const todos = [
+  {
+    description: 'Go to school',
+    completed: false,
+    index: 0,
+  },
+
+  {
+    description: 'cook',
+    completed: true,
+    index: 0,
+  },
+
+  {
+    description: 'wash',
+    completed: false,
+    index: 0,
+  },
+];
+
+todos.forEach((todo) => {
+  const p = document.createElement('p');
+  p.innerText = `${todo.description}`;
+});
```

Comment:
- Ensure that the following is worked on kindly ⬇️ 
- [ ] There is an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\].
- [ ] It has a function that iterates over the tasks array and populates the HTML list item element for each task. for example
```
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

- [ ] On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.
- [ ] Lastly, don't forget to use ES6 syntax and that means having `arrow functions` instead of the `function` keyword.